packages/flutter_tools/lib/src/dart/package_map.dart

```diff
@@ -11,6 +12,10 @@ import '../base/common.dart';
 import '../base/file_system.dart';
 import '../base/logger.dart';
 
+// No touching!
+@visibleForTesting
+bool debugDangerousIgnorePackageLoadErrors = false;
```

Comment:
🤕

Reply:
We don't actually depend on the packageConfig values for the unit test. The correct change is probably to make the loader an injectable class that could be mocked out for testing. This would be a temporary fix so that we can start shuffling tests.

Open to alternative approaches too

Reply:
Based on discussion, I'm going to keep pulling on this string by

1. Checking if this reproduces with a simpler test case.

2. Making the package loader injectable

Reply:
OK! turned out to be a similar issues to FsUtils